designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt

```diff
@@ -114,7 +114,7 @@ private interface DesignSwitcher {
 
     @Composable
     fun CustomComponent(
-        modifier: Modifier = Modifier,
+        modifier: Modifier, // = Modifier,
```

Comment:
Not sure why I needed this -- Compose compiler was complaining about Composables not being allowed to have defaults -- perhaps I need to upgrade Android Studio? (This is in main and CI is happy).

Reply:
I've seen this when doing inheritance, but yea don't know why this is needed.

Reply:
I may have found your answer, though I don't know why you only just started running into it: https://b.corp.google.com/issues/322121224

Compose Compiler 1.5.8 actually blocks the use of default values for open Composables entirely since they don't have support for it: https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.8

Reply:
But there's no inheritance going on, and all the generated functions from annotations look like this, with the first parameter being `modifier: Modifier = Modifier`.

Reply:
I deleted the commented code, since it's an internal implementation detail and not an externally exposed API.